browser-es6.js

```diff
@@ -21,5 +21,12 @@ module.exports = merge(base, {
   },
   plugins: [
     'react'
-  ]
+  ],
+  rules: {
+    'new-cap': 0,
+    'no-extra-parens': 0,
```

Comment:
It turns out that the React plugin does have a rule: `react/wrap-multilines`

Unfortunately, the `no-extra-parens` rule _still_ throws an error since one rule cannot just silence another. See yannickcr/eslint-plugin-react#86 for more information.

As such, I will propose we keep this off for ES6 code, but I don't want to move it into the base config.